app/views/pages/login.html.slim

```diff
@@ -10,8 +10,8 @@
         .card-body
           .container
             .form-group
-              button.btn.btn-facebook.btn-block Facebook
+              = link_to "Facebook", user_facebook_omniauth_authorize_path, class: "btn btn-facebook btn-block"
```

Comment:
__[Goodcheck]__

Misspelling?

GitHub is not Github.
JavaScript is not Javascript.
Facebook is not facebook!


<!-- Comment by [SideCI](https://sideci.com) -->